src/Translator/src/TranslationsDumper.php

```diff
@@ -94,6 +107,12 @@ private function getTranslations(MessageCatalogueInterface ...$catalogues): arra
         foreach ($catalogues as $catalogue) {
             $locale = $catalogue->getLocale();
             foreach ($catalogue->getDomains() as $domain) {
+                if (\in_array($domain, $this->excludedDomains, true)) {
+                    continue;
+                }
+                if (0 !== \count($this->includedDomains) && !\in_array($domain, $this->includedDomains, true)) {
```

Comment:
Can be simplified to:
```suggestion
                if ($this->includedDomains && !\in_array($domain, $this->includedDomains, true)) {
```